mpt_extension_sdk/api/auth/__init__.py

```
@@ -0,0 +1,10 @@
+from mpt_extension_sdk.api.auth.context import Account, AccountType, AuthContext
+from mpt_extension_sdk.api.auth.service import AuthenticationError, RequestAuthenticationService
+
+__all__ = [  # noqa: WPS410
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS410

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)